utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java

```diff
@@ -146,31 +148,41 @@ public void pullOperatorImage() {
      */
     public void grantPermissions() {
         log.info("Granting permissions to user {}", TestConfiguration.syndesisUsername());
-        new File(OpenShiftUtils.binary().getOcConfigPath()).setReadable(true, false);
 
         try {
-            new ProcessBuilder("docker",
-                "run",
-                "--rm",
-                "-v",
-                OpenShiftUtils.binary().getOcConfigPath() + ":/tmp/kube/config:z",
-                "--entrypoint",
-                "syndesis-operator",
-                TestConfiguration.syndesisOperatorImage(),
+            executeOperatorCommand(
                 "grant",
                 "-u",
-                TestConfiguration.syndesisUsername(),
-                "--namespace",
-                TestConfiguration.openShiftNamespace(),
-                "--config",
-                "/tmp/kube/config"
-            ).start().waitFor();
+                TestConfiguration.syndesisUsername()
+            );
         } catch (Exception e) {
             log.error("Unable to grant permissions", e);
             fail("Unable to grant permissions from docker run");
         }
     }
 
+    public void executeOperatorCommand(String... param) throws InterruptedException, IOException {
```

Comment:
I would vote for not throwing any exception here, rather handle them in the method

Reply:
also please change all the other methods to use this method as well

Reply:
@avano  Ok, I didn't use it in the other methods (`OperationValidationSteps#performRestore` and `Syndesis#getOperatorResources`) because the docker command looks little different (e.g. getOperatorResources doesn't use some parameters which I have in the `executeOperatorCommand` method)
Ok, I little bit refactor it and create new commit to see the changes, If you are ok with that, I merge it as one.

Reply:
https://github.com/syndesisio/syndesis-qe/pull/1326/commits/99c2f81bb668e778c507d216c6a4cf7cc2727c99